siege_utilities/geo/gazetteers/wikidata_gazetteer.py

```
@@ -0,0 +1,314 @@
+            )
+        query = _SPARQL_TEMPLATE % (name, limit)
+        try:
+            resp = self._session.get(
+                self._sparql_url,
+                params={"query": query, "format": "json"},
+                timeout=self._timeout,
+            )
+        except requests.exceptions.RequestException as exc:
+            raise GazetteerBackendError(
+                f"Wikidata SPARQL request for {name!r} failed: {exc}"
+            ) from exc
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)